src/lib/Form/Type/SiteAccessChoiceType.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\User\Form\Type;
+
+use Ibexa\Core\MVC\Symfony\SiteAccess\SiteAccessServiceInterface;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\ChoiceList\ChoiceList;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+final class SiteAccessChoiceType extends AbstractType
```

Comment:
Tbh it's not a User related form type. This could be extracted to core. @alongosz ?

This may or may not relate to other form types in this PR.

Reply:
Core does not include a single Type and I think it should stay this way. Even more, Siteaccesses are not part of the contracts there, so I would rather leave them here as there is no good place for that ans SectionsChoiceType. I will just move them to `Invitation` namespace to be clear about the intent.

Reply:
I don't mind adding a form type, but it seems generic, that's all. Ability to select site access is not limited to `ibexa/user` package.

Reply:
Totally agree, there is just no _good_ place for it atm in my opinion. (aside from adminUI maybe, but we have reversed dependency here).